fed/mosaic-mapping/src/main/java/org/eclipse/mosaic/fed/mapping/config/units/CChargingStation.java

```diff
@@ -46,6 +48,16 @@ public static class CChargingSpot {
          * The maximal current this charging spot can deliver.
          */
         public double maxCurrent;
+
+        @Override
+        public String toString() {
+            // NO_CLASS_NAME_STYLE for better formatting in ChargingStationSpawner
+            return new ToStringBuilder(this, ToStringStyle.NO_CLASS_NAME_STYLE)
```

Comment:
SHORT_PREFIX_STYLE

Reply:
this makes the output of the `ChargingStationSpawner` quite ugly as the "short prefix" would translate to `CChargingStation.CChargingSpot[...]`

Reply:
How does it look with `NO_CLASS_NAME_STYLE`, just `CChargingSpot[...]`?

Reply:
No just square brackets: `[...]`

Output can look like this:
```
INFO  ChargingStationSpawner - Creating Charging Station: ChargingStationSpawner[protoTypeName=<null>,group=chargingStation,applications=[com.dcaiti.mosaic.test.app.chargingstationtest.ChargingStationApp],position=GeoPoint{lat=52.512841,lon=13.328037,alt=0.00},chargingSpots=[chargingSpot(0)=[chargingType=AC_3_PHASE,maxVoltage=400.0,maxCurrent=16.0], chargingSpot(1)=[chargingType=AC_3_PHASE,maxVoltage=400.0,maxCurrent=16.0]]]

```